Podfile.lock

```diff
@@ -1,46 +0,0 @@
-PODS:
```

Comment:
Why delete the Podfile.lock? Was this a mistake?

Reply:
I was getting a CI cocoapods error where it said it couldn't find 3.6.0 so i wanted to see if removing the podfile.lock would fix it. And it did.

Reply:
done